java/sidecar-common/src/main/java/com/instaclustr/sidecar/operations/OperationRequest.java

```diff
@@ -14,7 +14,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public abstract class OperationRequest {
 
-    @JsonProperty("type")
+    @JsonIgnore
```

Comment:
no worries, this will be still there because it is "type" info

Reply:
Why does request even have a "type" field, and how does this conflict with the `@JsonTypeInfo` property?

Reply:
The type of a request _is its class type_.